src/components/File.scss

```diff
@@ -0,0 +1,16 @@
+.file__container {
+  padding-top: 1rem;
+
+  .languages {
```

Comment:
Using the BEM convention in the JSX, you'd end up with a CSS selection tree like this:

```css
.file {
  &__languages {
    ...
  }
}
```